src/bootstrap.js

```diff
@@ -14,10 +18,25 @@ function install() {} // eslint-disable-line no-unused-vars
 function uninstall() {} // eslint-disable-line no-unused-vars
 
 function startup() { // eslint-disable-line no-unused-vars
+  Services.prefs.setBoolPref(UA_ENABLE_PREF_NAME, true);
```

Comment:
I prefer not to enable preference in code, even if it's just in development stage. We should check preference then init UAOverrider, also add observer for changes.